packages/vue-app/src/index.js

```diff
@@ -1,26 +1,26 @@
 import path from 'path'
-import pkg from '../package.json'
+import { dependencies } from '../package.json'
 
-export const meta = pkg
-
-export const templatesDir = path.join(__dirname, '..', 'template')
-
-export const templatesFiles = [
-  'App.js',
-  'client.js',
-  'index.js',
-  'middleware.js',
-  'router.js',
-  'server.js',
-  'utils.js',
-  'empty.js',
-  'components/nuxt-error.vue',
-  'components/nuxt-loading.vue',
-  'components/nuxt-child.js',
-  'components/nuxt-link.server.js',
-  'components/nuxt-link.client.js',
-  'components/nuxt.js',
-  'components/no-ssr.js',
-  'views/app.template.html',
-  'views/error.html'
-]
+export const template = {
```

Comment:
Should this export be default ?

Reply:
Yes. But it breaks `yarn dev` script as we do `export * from '@nuxt/vue-app'`. Also reserving namespace for other top-level keys like `version` and `manifest`

Reply:
gotcha